### PR TITLE
Add support for setting default mapping ID

### PR DESCRIPTION
**Pull request by hotzenklotz**
### Description:
- PR adds the option to specify the default segmentation layer mapping through the libs. 
- See also parent [WK PR](https://github.com/scalableminds/webknossos/pull/7858)

```
import webknossos as wk
from webknossos.dataset.properties import LayerViewConfiguration

d = wk.Dataset(some_path)
s = d.get_segmentation_layer()

default_mapping = {"name": "agglomerate_view_75", "type": "hdf5"}
s.default_view_configuration = LayerViewConfiguration(mapping=default_mapping)
```

### Issues:
- related to https://github.com/scalableminds/webknossos/pull/7858

### Todos:
Make sure to delete unnecessary points or to check all before merging:
 - [x] Updated Changelog
 - [x] Updated Documentation
 - [ ] Added / Updated Tests
 - [ ] Considered adding this to the Examples

### CODE REVIEW

File: webknossos/webknossos/dataset/properties.py [hunk 1]
```diff
@@ -113,6 +113,9 @@ class LayerViewConfiguration:
     is_in_edit_mode: Optional[bool] = None
     """Enable the histogram edit mode. The WEBKNOSSOS default is False."""
 
+    mapping: Optional[Dict[str, str]] = None
+    """Enables ID mapping for a segmentation layer and applies the selected mapping by default. The default WK behavior is to disable ID mapping. Expected values is a Dict with {"name": my_mapping_name, "type": "hdf5"}."""
```
Comment (fm3) [inline webknossos/webknossos/dataset/properties.py:117]: @MichaelBuessemeyer do you know if the type needs to be written in capital letters here, HDF5?

Comment (MichaelBuessemeyer) [reply webknossos/webknossos/dataset/properties.py:117]: Good catch. It needs to be in caps. See
https://github.com/scalableminds/webknossos/blob/c50a81860b5e9882b24677e5a750b10f63c5d6a8/frontend/javascripts/oxalis/store.ts#L396

Comment (fm3) [reply webknossos/webknossos/dataset/properties.py:117]:
```suggestion
    """Enables ID mapping for a segmentation layer and applies the selected mapping by default. The default WK behavior is to disable ID mapping. Expected values is a Dict with {"name": my_mapping_name, "type": "HDF5"}."""
```

**Review by fm3 — APPROVED** [review]
LGTM

**Pull request merged**